arch/arm/src/armv8-m/arm_doirq.c

```diff
@@ -67,6 +67,21 @@ void exception_direct(void)
     }
 }
 
+static inline_function bool store_context(int irq, uint32_t *regs)
```

Comment:
I'm really tempted to complain that arm_doirq used to be a piece of extremely concise code, with clear logic and easy readability. But now, the modifications made to it are truly terrible ...